challenge/public/index.html

```diff
@@ -13,5 +13,57 @@
 <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <style>
```

Comment:
after I merged in master, you should now be able to move this to `public/styles.css`